test/cbmc/proofs/OTA_CBOR_Decode_GetStreamResponseMessage/OTA_CBOR_Decode_GetStreamResponseMessage_harness.c

```diff
@@ -0,0 +1,107 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/**
+ * @file OTA_CBOR_Decode_GetStreamResponseMessage_harness.c
+ * @brief Implements the proof harness for OTA_CBOR_Decode_GetStreamResponseMessage function.
+ */
+/* Include headers for cbor parsing. */
+#include "cbor.h"
+#include "ota_cbor_private.h"
+
+/* Stub to initialize the cbor parser. */
+CborError cbor_parser_init( const uint8_t * buffer,
+                            size_t size,
+                            uint32_t flags,
+                            CborParser * parser,
+                            CborValue * it )
+{
+    CborError err;
+
+    return err;
+}
```

Comment:
NIT: Can apply to all functions below:
```suggestion
{
   returnCborError();
}

CborError returnCborError( )
{
    CborError err;
    return err;
}
```